docker/Dockerfile

```diff
@@ -376,6 +380,13 @@ RUN --mount=type=bind,from=build,src=/workspace/dist,target=/vllm-workspace/dist
     uv pip install --system dist/*.whl --verbose \
         --extra-index-url ${PYTORCH_CUDA_INDEX_BASE_URL}/cu$(echo $CUDA_VERSION | cut -d. -f1,2 | tr -d '.')
 
+# TODO (huydhn): Remove this once xformers is released for 2.8.0
+RUN --mount=type=cache,target=/root/.cache/uv bash - <<'BASH'
+    . /etc/environment
+    export TORCH_CUDA_ARCH_LIST='7.0 7.5 8.0 8.9 9.0 10.0 12.0'
+    uv pip install --system --no-build-isolation "git+https://github.com/facebookresearch/xformers@v0.0.30"
```

Comment:
Is there a reason why you aren't using [v0.0.31](https://github.com/facebookresearch/xformers/releases/tag/v0.0.31)? I previously upgraded this intentionally for 2.7.1 https://github.com/vllm-project/vllm/commit/4e7dfbe7b49a3386e3212209fcc093c0c166ba95#diff-f0bb597ba1276975a33aef58f30237934c3fa54f627f5020210df7301acba61a